src/MailViewerServiceProvider.php

```diff
@@ -15,6 +16,15 @@ public function boot()
         $this->loadRoutesFrom(__DIR__ . '/../routes/web.php');
 
         $this->loadViewsFrom(__DIR__ . '/../views', 'mailviewer');
+
+        if (app()->environment() === 'testing') {
```

Comment:
Curious to know what is the benefit of this container binding since it might effect user's own testing env

Reply:
I had to do this to make sure the `TestFactory.php` class would be loaded when running the unit tests. Laravel sets the default environment to `testing` when running unit tests so shouldn't cause a problem for anyone's testing env. See [the code on Laravel's repo](https://github.com/laravel/framework/blob/763b64a43ebb6042e463aab4214d4cc9722147be/src/Illuminate/Foundation/Application.php#L530).